engine/runtime/src/test/java/org/enso/compiler/CompilerTest.java

```diff
@@ -31,4 +37,82 @@ protected static IR.Module parse(String code) throws IOException {
     assertNotNull("IR was generated", ir);
     return ir;
   }
+
+  static void assertIR(String msg, IR.Module old, IR.Module now) throws IOException {
+    Function<IR, String> filter = f -> simplifyIR(f, true, true, false);
+    String ir1 = filter.apply(old);
+    String ir2 = filter.apply(now);
+    if (!ir1.equals(ir2)) {
+      String name = findTestMethodName();
+      Path home = new File(System.getProperty("user.home")).toPath();
+      Files.writeString(home.resolve(name + ".1"), ir1, StandardOpenOption.TRUNCATE_EXISTING, StandardOpenOption.CREATE, StandardOpenOption.WRITE);
```

Comment:
Can we try and not write those to a user's home directory? A directory inside a checked out repo that is simply ignored would be much better

Reply:
The files are written only in case of failure and are really helpful when one needs to `diff` them. I was rewriting the old parser to new one without the files being dump and it was plain horrible - e.g. ignoring the files isn't an option. Feel free to suggest better location instead of `home`. Anyway, this is not code introduced by this PR - I have just refactored it to another class.

Reply:
Alternatively, it can write to the `tmp` directory

Reply:
@JaroslavTulach I know. I didn't like it then, I don't like it now. I understand the sentiment that with hundreds failures you really want to have something nearby but I also don't like things polluting my home directory.